client/client.go

```diff
@@ -59,7 +59,8 @@ func (conn *Connection) call(ctx context.Context, reqs []interface{}, resp inter
 	if err != nil {
 		return nil, err
 	}
-	return conn.refOpts.invoker.Invoke(ctx, inv), nil
+	res := conn.refOpts.invoker.Invoke(ctx, inv)
```

Comment:
It seems there is no need to change statement style.

Reply:
Yes, the change here is a mistake. This was a change while testing, I will change it back to how it was.